src/components/layout/header.js

```diff
@@ -35,7 +36,7 @@ const Header = ({ siteTitle }) => (
                     padding: `0.5rem`,
                 }}
             >
-                Home
+                <Localize i18n_text="Home" />
```

Comment:
what an ugly syntax 😃

Reply:
Got a more beautiful suggestion? 😅 Either way can change it later when we add translations

Reply:
Maybe just `text` instead of `i18n_text`?

Reply:
@zaki yeah `text` is cleaner but I think we want to keep it more specific incase we need a extract translations script?

Reply:
`translate`?